views/default/forms/admin/users/search.php

```diff
@@ -15,9 +15,10 @@
 	'#type' => 'fieldset',
 	'fields' => [
 		[
-			'#type' => 'text',
+			'#type' => 'search',
 			'#class' => 'elgg-field-stretch',
 			'name' => 'q',
+			'aria-label' => elgg_echo('search'), // because we don't have a #label
 			'placeholder' => elgg_echo('search'),
```

Comment:
should we magically default aria-label to the placeholder? this feels like a lot of duplication